test/core/crypto/blake2/blake2_test.cpp

```diff
@@ -0,0 +1,123 @@
+#include <gtest/gtest.h>
```

Comment:
No header. Is that file copied from somewhere? If so we can leave it as is. Otherwise please also add given when then descriptions

Reply:
Yes, the file is copied from the same repo as the Blake2 code itself